### PR TITLE
chore(dependencies): Update all dependencies

### PR DESCRIPTION
**Pull request by daffl**

### CODE REVIEW

File: packages/socketio/package.json [hunk 1]
```diff
@@ -53,20 +53,20 @@
     "@feathersjs/commons": "^5.0.0-pre.15",
     "@feathersjs/feathers": "^5.0.0-pre.15",
     "@feathersjs/transport-commons": "^5.0.0-pre.15",
-    "socket.io": "^4.3.1"
+    "socket.io": "^4.4.0"
   },
   "devDependencies": {
     "@feathersjs/commons": "^5.0.0-pre.2",
```
Comment (8BallBomBom) [inline packages/socketio/package.json]: Still got a potential issue here line 59.
Supposed to be removed as it is already in dependencies?

Comment (daffl) [reply packages/socketio/package.json]: Ah, I was confused before because I thought it was about the `socket.io` dependency change.

Comment (daffl) [reply packages/socketio/package.json]: Good catch, fixed now. Thanks!

Comment (8BallBomBom) [reply packages/socketio/package.json]: No worries, the line comments do seem to show a bit weird sometimes haha 👍🏻

**Pull request merged**